client/src/components/Applications/AppForm/AppForm.jsx

```diff
@@ -660,7 +660,7 @@ export const AppSchemaForm = ({ app }) => {
                         description="Directory into which output files are archived after application execution."
                         name="archiveSystemDir"
                         type="text"
-                        placeholder="HOST_EVAL($WORK)/tapis-jobs-archive/${JobCreateDate}/${JobName}-${JobUUID}" // TODOv3: How do we know if portal supports HOME vs WORK?
+                        placeholder="HOST_EVAL($WORK)/tapis-jobs-archive/${JobCreateDate}/${JobName}-${JobUUID}"
```

Comment:
Actually I think we may want to keep this comment and task. The task should be to determine a safe root path from which we can archive. Perhaps using the `homeDir` of the `defaultSystem`

Reply:
I'll add that back in and create a task for it as well

Reply:
Created [WP-103](https://jira.tacc.utexas.edu/browse/WP-103)